packages/react/src/components/Tree/TreeItem.tsx

```diff
@@ -78,20 +90,24 @@ class TreeItem extends AutoControlledComponent<ReactProps<TreeItemProps>, TreeIt
     accessibility: defaultBehavior,
   }
 
+  public state = {
+    open: false,
+  }
```

Comment:
As this component should be autocontrolled, please use `getInitialAutoControlledState()`

```jsx
  getInitialAutoControlledState({ multiple, search }: DropdownProps): DropdownState {
    return {
      open: false,
    }
  }
```